src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import 'jest-enzyme';
+
+configure({ adapter: new Adapter() });
```

Comment:
Do you think you could add a few comments here?

Reply:
Definitely @Karuhanga. I believe it will add more clarity. Let me do it